src/TConsole.cpp

```diff
@@ -442,7 +448,8 @@ TConsole::TConsole(Host* pH, bool isDebugConsole, QWidget* parent)
     emergencyStop->setSizePolicy(sizePolicy4);
     emergencyStop->setFocusPolicy(Qt::NoFocus);
     emergencyStop->setCheckable(true);
-    emergencyStop->setToolTip(tr("<html><head/><body><p>Emergency Stop. Stops all timers and triggers.</p></body></html>"));
+    emergencyStop->setToolTip(
```

Comment:
Is this line being reformatted just because of a `clang-format` action?

I'm asking because it will cause a conflict with my pending #1656 which also disables Key-bindings when the Emergency-Stop is hit and thus now gets included in this text {which I added because I found during testing that it is essential if you forget to use a modifier for a normal key as then you cannot enter that key into the command line! 😮 }

Reply:
Yes - I've undone the change.